stage1/init/kvm/hypervisor/hypervisor.go

```diff
@@ -44,4 +46,9 @@ func (hv *KvmHypervisor) InitKernelParams(isDebug bool) {
 			"quiet systemd.log_level=emerg",
 		}...)
 	}
+
+	customKernelParams := os.Getenv("RKT_HYPERVISOR_CUSTOM_KERNEL_PARAMS")
```

Comment:
Did you try with multiple space-separated arguments in here? I fear they will end up coalesced as a single space-embedding parameter from exec() point of view.